scripts/shared/authDataLoader.js

```diff
@@ -0,0 +1,48 @@
+/*!
```

Comment:
I think I'd prefer to see all the contents of scripts/shared placed into a fresh "micromodule" named dataLoader, and all the tests can then be similarly named, since tests/scripts looks rather peculiar